documentation/smg/02_development_environment.md

```diff
@@ -213,5 +213,28 @@ At the time of writing, the only way I know to get these onto your laptop is to
 I use WinSCP. The user id and password are the same as for getting into ISIS Analysis (your STFC email and your chosen password),
 and the hostname can be found in the window title bar of the VM browser window -
 **NOT** the local name starting `host-192...` which you see in the individual terminal windows.
-Port is the default 22. An alternative tool is rsync, which I have not tried.
+Port is the default 22. An alternative tool is rsync, which I have not tried. 
+
+The Scratch location for this directory is not ideal, and to provide a more permanent home, 
```

Comment:
Rather than refer to both the scratch location and the new permanent home on the SAN drive, these instructions should simply describe the permanent home of the data and how to access it there.

Reply:
Not at present. Anyone currently wishing to access the demo on IDaaS cannot get access to the SAN disk and will need a local copy. While that copy is available it is worth mentioning for ease of use. They will not want to go through the windows server get it from SAN to my scratch only to discover it was there all along.